docs/mkdocs/docs/technical/contributing.md

```diff
@@ -1,19 +1,88 @@
-Setting up your OS dependencies
-===============================
+Docker Quickstart
+=================
 
-ManyLinux Docker image
-----------------------
-ManyLinux(https://github.com/pypa/manylinux) captures the common denominator of many Linux distributions,
-giving the confidence that code worked in this image runs everywhere.
+#### 1) Start the ArcticDB build docker image
 
-We [extend](https://github.com/man-group/ArcticDB/blob/master/build_tooling/build_many_linux_image.sh) it with
-additional dependencies for compilation purposes.
-You can download the output [here](https://github.com/man-group/ArcticDB/pkgs/container/cibuildwheel_manylinux).
+```bash
+docker pull ghcr.io/man-group/cibuildwheel_manylinux:2.12.1-3a897
+docker run -it ghcr.io/man-group/cibuildwheel_manylinux:2.12.1-3a897
+```
+
+> :warning: The below instructions do not have to be run in the provided docker image. They can be run against any Python installation on any Linux distribution as long as the basic build [dependencies are available](#setting-up-linux).
+> 
+> If running outside the provided docker image, please change `/opt/python/cp39-cp39/bin/python3` in the examples below 
+> to an appropriate path for Python.
+
+#### 2) Check out ArcticDB including submodules
+
+```bash
+cd 
+git clone https://github.com/man-group/ArcticDB.git
+cd ArcticDB
+git submodule init && git submodule update
+```
+
+#### 3) Kick off the build
+
+```bash
+/opt/python/cp39-cp39/bin/python3 -m pip install -U pip setuptools wheel grpcio-tools
+ARCTIC_CMAKE_PRESET=skip /opt/python/cp39-cp39/bin/python3 setup.py develop
+# Change the below Python_EXECUTABLE value to build against a different Python version
+cmake -DPython_EXECUTABLE=/opt/python/cp39-cp39/bin/python -DTEST=off --preset linux-debug cpp
+# NOTE: Set parallel level below to number of cores!
+cmake --build --parallel 8 --preset linux-debug cpp
+```
+
+#### 4) Run ArcticDB
+
+Ensure the below is run in the Git project root:
+
+```bash
+# PYTHONPATH first part = Python module, second part compiled C++ binary
+PYTHONPATH=`pwd`/python:`pwd`/cpp/out/linux-debug-build/arcticdb/ /opt/python/cp39-cp39/bin/python3
```

Comment:
As long as `ARCTIC_CMAKE_PRESET=skip` is not set, `pip install/setup.py` will build and copy the module to the correct location. It's not necessary to fiddle with the `PYTHONPATH`.